src/Entity/Task/RecurrenceRule.php

```diff
@@ -16,47 +23,35 @@
 use Symfony\Component\Validator\Constraints as Assert;
 
 #[ApiResource(
-    collectionOperations: [
-        'get' => [
-            'method' => 'GET',
-            'security' => "is_granted('ROLE_ADMIN') or is_granted('ROLE_DISPATCHER')"
-        ],
-        'post' => [
-            'method' => 'POST',
-            'security' => "is_granted('ROLE_ADMIN') or is_granted('ROLE_DISPATCHER')"
-        ],
-        'generate_orders' => [
-            'method' => 'GET',
-            'path' => '/recurrence_rules/generate_orders',
-            'security' => "is_granted('ROLE_DISPATCHER')",
-            'controller' => GenerateOrders::class,
-            'status' => 201
-        ]
-    ],
-    itemOperations: [
-        'get' => [
-            'method' => 'GET',
-            'security' => "is_granted('ROLE_ADMIN') or is_granted('ROLE_DISPATCHER')"
-        ],
-        'put' => [
-            'method' => 'PUT',
-            'security' => "is_granted('ROLE_ADMIN') or is_granted('ROLE_DISPATCHER')"
-        ],
-        'between' => [
-            'method' => 'POST',
-            'path' => '/recurrence_rules/{id}/between',
-            'security' => "is_granted('ROLE_ADMIN') or is_granted('ROLE_DISPATCHER')",
-            'controller' => BetweenController::class
-        ],
-        'delete' => [
-            'method' => 'DELETE',
-            'security' => "is_granted('ROLE_ADMIN') or is_granted('ROLE_DISPATCHER')"
-        ]
-    ],
     shortName: 'RecurrenceRule',
+    operations: [
+        new Get(
+            requirements: ['id' => '[0-9]+'],
+            // Make sure to add requirements for operations like "/recurrence_rules/generate_orders" to work
```

Comment:
It seems something else is still needed. I get:

![Screenshot 2025-05-07 at 16 39 55](https://github.com/user-attachments/assets/5be0d384-d4c5-4d30-b664-ae672e432dea)

Reply:
Fixed in https://github.com/coopcycle/coopcycle-web/pull/4964/commits/66571e1cc7ac4948c42c89f3a6f71f7848a944bc